src/cascadia/CascadiaResources.build.items

```diff
@@ -1,248 +1,29 @@
-      <Link>Images\Square44x44Logo.targetsize-72.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Square44x44Logo.targetsize-72_altform-unplated.png">
-      <Link>Images\Square44x44Logo.targetsize-72_altform-unplated.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Square44x44Logo.targetsize-80.png">
-      <Link>Images\Square44x44Logo.targetsize-80.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Square44x44Logo.targetsize-80_altform-unplated.png">
-      <Link>Images\Square44x44Logo.targetsize-80_altform-unplated.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Square44x44Logo.targetsize-96.png">
-      <Link>Images\Square44x44Logo.targetsize-96.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Square44x44Logo.targetsize-96_altform-unplated.png">
-      <Link>Images\Square44x44Logo.targetsize-96_altform-unplated.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\StoreLogo.scale-100.png">
-      <Link>Images\StoreLogo.scale-100.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\StoreLogo.scale-125.png">
-      <Link>Images\StoreLogo.scale-125.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\StoreLogo.scale-150.png">
-      <Link>Images\StoreLogo.scale-150.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\StoreLogo.scale-200.png">
-      <Link>Images\StoreLogo.scale-200.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\StoreLogo.scale-400.png">
-      <Link>Images\StoreLogo.scale-400.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Wide310x150Logo.scale-100.png">
-      <Link>Images\Wide310x150Logo.scale-100.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Wide310x150Logo.scale-125.png">
-      <Link>Images\Wide310x150Logo.scale-125.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Wide310x150Logo.scale-150.png">
-      <Link>Images\Wide310x150Logo.scale-150.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Wide310x150Logo.scale-200.png">
-      <Link>Images\Wide310x150Logo.scale-200.png</Link>
-    </Content>
-    <Content Include="$(OpenConsoleDir)res\terminal\Wide310x150Logo.scale-400.png">
-      <Link>Images\Wide310x150Logo.scale-400.png</Link>
+    <!-- Images -->
+    <Content Include="$(OpenConsoleDir)res\terminal\**\*">
+      <DeploymentContent>true</DeploymentContent>
+      <Link>Images\%(RecursiveDir)%(FileName)%(Extension)</Link>
     </Content>
     <!-- Fonts -->
     <Content Include="$(OpenConsoleDir)res\Cascadia.ttf" Condition="'$(WindowsTerminalReleaseBuild)'=='true'">
-      <Link>Cascadia.ttf</Link>
+      <DeploymentConent>true</DeploymentConent>

```

Comment:
we confirmed the ttf ended up in the package, yeah?

Reply:
it didn't because I typo'd this as "DeploymentCo**ne**nt". If I spell it correctly, it's fine.